sdk/metric/doc.go

```diff
@@ -97,21 +69,19 @@ enters the SDK resulting in a new record, and collection context,
 where a system-level thread performs a collection pass through the
 SDK.
 
-Descriptor is a struct that describes the metric instrument to the export
-pipeline, containing the name, recommended aggregation keys, units,
-description, metric kind (counter or measure), number kind (int64 or
-float64), and whether the instrument has alternate semantics or not (i.e.,
-monotonic=false counter, absolute=false measure).  A Descriptor accompanies
-metric data as it passes through the export pipeline.
+Descriptor is a struct that describes the metric instrument to the
+export pipeline, containing the name, recommended aggregation keys,
+units, description, metric kind, number kind (int64 or float64).  A
+Descriptor accompanies metric data as it passes through the export
+pipeline.
```

Comment:
No recommended keys anymore.

```suggestion
export pipeline, containing the name, units, description, metric kind, and
number kind (int64 or float64). A Descriptor accompanies metric data as it
passes through the export pipeline.
```

Reply:
Done.